specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations.json

```diff
@@ -2,1120 +2,915 @@
   "swagger": "2.0",
   "info": {
     "title": "Microsoft Cognitive Language Service - Analyze Conversations",
-    "description": "The language service conversations API is a suite of natural language processing (NLP) skills that can be used to analyze structured conversations (textual or spoken). The synchronous API in this suite accepts a request and mediates among multiple language projects, such as LUIS Generally Available, Question Answering, Conversational Language Understanding, and then calls the best candidate service to handle the request. At last, it returns a response with the candidate service's response as a payload.\n\n In some cases, this API needs to forward requests and responses between the caller and an upstream service. The asynchronous APIs in this suite enable tasks like Conversation Summarization and Conversational PII detection.",
-    "version": "2022-05-01"
-  },
-  "securityDefinitions": {
-    "apim_key": {
-      "type": "apiKey",
-      "description": "A subscription key for a Language service resource.",
-      "name": "Ocp-Apim-Subscription-Key",
-      "in": "header"
-    }
+    "version": "2022-05-01",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
   },
-  "security": [
-    {
-      "apim_key": []
-    }
+  "schemes": [
+    "https"
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/language",
+    "hostTemplate": "{endpoint}/language",
     "useSchemePrefix": false,
     "parameters": [
       {
-        "$ref": "common.json#/parameters/Endpoint"
+        "name": "endpoint",
+        "in": "path",
+        "required": true,
+        "description": "Supported Cognitive Services endpoint (e.g., https://<resource-name>.api.cognitiveservices.azure.com).",
+        "type": "string"
       }
     ]
   },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "description": "A subscription key for a Language service resource",
+      "in": "header",
+      "name": "Ocp-Apim-Subscription-Key"
+    }
+  },
+  "tags": [],
   "paths": {
-    "/:analyze-conversations": {
+    "/language/:analyze-conversations": {
```

Comment:
This shouldn't be here. It's already the default for the endpoint (see `hostTemplate` above).

Reply:
In general, @johanste or @lmazuel, for brownfield services, do we want them overwriting swaggers especially when we're going for "payload compatibility" instead of 1:1 similarity?

Reply:
Yeah, that's a good idea. Go ahead and remove it from the `hostTemplate`. @johanste, @bterlson what is the correct way to set a "root" URI in an endpoint and not having to repeat it as Zach has had to do here? All Language pillar endpoints will begin with "/language", so it seems more idiomatic to define it once.